consent/strategy_default_test.go

```diff
@@ -181,7 +181,7 @@ func logoutHandler(strategy Strategy, writer herodot.Writer, w http.ResponseWrit
 	http.Redirect(w, r,
 		urlx.CopyWithQuery(
 			urlx.ParseOrPanic(res.RedirectTo),
-			url.Values{"front_channel_logout_urls": {fmt.Sprintf("%s", res.FrontChannelLogoutURLs)}},
```

Comment:
I removed this because it made the assertions easier (and I don't know what it was for).

Reply:
Hm strange, looks like this isn't used anywhere. Maybe it was there for an older test case, it should be ok to remove.